testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientClientScopesTest.java

```diff
@@ -98,7 +98,7 @@ public void testSetupClientScopes() {
 
         // Retrieve client through adminClient
         found = findClientByClientId(TEST_CLIENT_ID);
-        Assert.assertNames(found.getDefaultClientScopes(), "email", "role_list"); // SAML client scope 'role_list' is included too in the rep
+        Assert.assertNames(found.getDefaultClientScopes(), "email"); // SAML client scope 'role_list' is included too in the rep
```

Comment:
The comment is irrelevant now, can you please remove it?

Reply:
Done